package.json

```diff
@@ -11,6 +11,7 @@
     "node": ">=5"
   },
   "scripts": {
+    "postinstall": "cd lighthouse-core && npm install && npm run build",
```

Comment:
I think @patrickhulce argued that `prepublish` is the way to go for this. That makes sure everything is built before it gets published to npm, users installing lighthouse via `npm install` don't have to do compilation work, and users installing via `git clone` who then `npm install` get `prepublish` called for them, so everything is built.

One issue is that we don't really want to push a built extension (with giant browserified bundle) to npm, but we could add an `.npmignore` file so e.g. the compiled js files in `cli/` get published but the compiled extension does not